src/new-client/src/App/LiveRates/LiveRates.tsx

```diff
@@ -16,15 +20,25 @@ const LiveRateWrapper = styled(Wrapper)`
   overflow-y: auto;
 `
 
-export const OverflowScroll = styled.div`
+const OverflowScroll = styled.div`
   overflow-y: scroll;
   height: 100%;
 `
 
+const liveRates$ = merge(tiles$, mainHeader$)
+
 export const LiveRates: React.FC = () => (
   <LiveRateWrapper>
     <OverflowScroll>
-      <MainPanel />
+      <div data-qa="workspace__tiles-workspace">
```

Comment:
Do you need three layers of wrapper? div/OverflowScrollLiveRateWrapper

Reply:
Of course not 🙂 .

However, the refactored code is (for now) supposed to produce the exact same DOM elements (and css styles) than we had before... (Unless there is a major reason not to).

FYI: once we have refactored all the services, feature-components, tests, etc We will look into properly leaveraging a better desing-system, but until then it's ok to keep producing the same DOM structure and css styles.